config/install.yaml

```diff
@@ -12266,7 +12266,6 @@ spec:
             weight: 1
       containers:
       - args:
-        - --filterK8sResources=[Event,*,*][*,kube-system,*][*,kube-public,*][*,kube-node-lease,*][Node,*,*][APIService,*,*][TokenReview,*,*][SubjectAccessReview,*,*][*,kyverno,kyverno*][Binding,*,*][ReplicaSet,*,*][ReportChangeRequest,*,*][ClusterReportChangeRequest,*,*][PolicyReport,*,*][ClusterPolicyReport,*,*]
```

Comment:
Is this container flag also present in Helm chart?

Reply:
No, but it's present in `config/release/install.yaml`. Shall I remove it from there ?

Reply:
Yes please, otherwise pulling from `config/release/install.yaml` on main will panic.

Reply:
Do you want me to add this change in `CHANGELOG.md` ?

Reply:
Done.

Reply:
It looks great, thanks!